pkg/uibackend/rest/dashboard_riskiest_assets.go

```diff
@@ -231,8 +231,15 @@ func containerInfoToAssetInfo(info backendmodels.ContainerInfo) (*models.AssetIn
 }
 
 func containerImageInfoToAssetInfo(info backendmodels.ContainerImageInfo) (*models.AssetInfo, error) {
-	repo, name := path.Split(*info.Name)
-	repo = strings.TrimSuffix(repo, "/")
+	var repo, name string
+
+	tag, ok := info.GetRepoTag()
+	if ok {
+		repo, name = path.Split(tag)
+	} else {
+		name = info.ImageID
+	}
```

Comment:
In the kubernetes work I handled this slightly differently: Name is always the ID, Location is the first repo digest (if there is one). The reason for using the whole digest as the location is because the repo is like the street name, but the digest is like the house number. Its an address where you can find the image with this ID if you curl it.

Reply:
Lets have the same logic as @Tehsmash discribed above.